workflows/dnaseq/dnaseq-core.wdl

```diff
@@ -0,0 +1,160 @@
+## **WARNING:** this workflow is experimental! Use at your own risk!
+
+version 1.1
+
+import "../../data_structures/read_group.wdl"
+import "../../tools/bwa.wdl"
+import "../../tools/picard.wdl"
+import "../../tools/samtools.wdl"
+import "../../tools/util.wdl"
+import "../general/bam-to-fastqs.wdl" as bam_to_fastqs_wf
+import "../general/samtools-merge.wdl" as samtools_merge_wf
+
+workflow dnaseq_core_experimental {
+    meta {
+        description: "Aligns DNA reads using bwa"
+        outputs: {
+            harmonized_bam: "Harmonized DNA-Seq BAM, aligned with bwa",
+            harmonized_bam_index: "Index for the harmonized DNA-Seq BAM file"
+        }
+        allowNestedInputs: true
+    }
+    parameter_meta {
+        read_one_fastqs_gz: "Input gzipped FASTQ format file(s) with 1st read in pair to align"
+        read_two_fastqs_gz: "Input gzipped FASTQ format file(s) with 2nd read in pair to align"
+        bwa_db: "Gzipped tar archive of the bwa reference files. Files should be at the root of the archive."
+        reads_per_file: "Number of reads per FASTQ file to output."
+        read_groups: "A string containing the read group information to output in the BAM file. If including multiple read group fields per-read group, they should be space delimited. Read groups should be comma separated, with a space on each side (i.e. ' , '). The ID field must come first for each read group and must be contained in the basename of a FASTQ file or pair of FASTQ files if Paired-End. Example: `ID:rg1 PU:flowcell1.lane1 SM:sample1 PL:illumina LB:sample1_lib1 , ID:rg2 PU:flowcell1.lane2 SM:sample1 PL:illumina LB:sample1_lib1`. These two read groups could be associated with the following four FASTQs: `sample1.rg1_R1.fastq,sample1.rg2_R1.fastq` and `sample1.rg1_R2.fastq,sample1.rg2_R2.fastq`"
```

Comment:
This needs to be updated to match that it takes `Array[ReadGroup]`. I think it's ok to stay on the terse side, as this isn't meant to be user facing.